content/departments/ce-support/ce/process/license_keys.md

```diff
@@ -10,6 +10,14 @@ For a new trial, create a _new_ subscription. You will only use this for the tri
 
 When the customer converts, create a _new_ subscription; do not reuse the trial subscription. (Follow the instructions below from start to finish.) This is to ensure data accuracy for the Sales Ops team.
 
+## How to reissue a license key for an expiring license
+
+In most circumstances, license keys match the renewal dates on an account contract and are generated well ahead of time. Sometimes, license keys deviate from this format due to product trials and other special circumstances. If a license key is imminently expiring or has expired, follow these steps:
+
+1. If possible, check with the account's Customer Engineer & Account Executive, as well as the account running notes. This will help you understand any special context surrounding the current license key tags and expiration dates. The account team may opt to extend a product trial key for a specific amount of time.
```

Comment:
Obviously best future case is the concept of a product trial is totally separate from the core Sourcegraph license, but let's work with what we have for now.

Do we want to add any additional instructions for the cases where it _is_ a code insights/batch changes trial? We ideally wouldn't want to accidentally add it to their annual license; but we don't have a standardized way of flagging this right now. (Maybe with Vitally we can? Or make everyone add this to their running notes in the meantime?)

Reply:
We will be moving to product specific subscriptions in the near future and away from this account level single license key structure. It does mean that we'd have licenses per product subscription so I think the guidance is more whatever license is expired... does that make sense?